opendevin/llm/bedrock.py

```diff
@@ -0,0 +1,26 @@
+import boto3
+
+from opendevin import config
+from opendevin.logger import opendevin_logger as logger
+from opendevin.schema import ConfigType
+
+AWS_ACCESS_KEY_ID = config.get(ConfigType.AWS_ACCESS_KEY_ID)
+AWS_SECRET_ACCESS_KEY = config.get(ConfigType.AWS_SECRET_ACCESS_KEY)
+AWS_REGION_NAME = config.get(ConfigType.AWS_REGION_NAME)
+
+
+def list_foundation_models():
+    try:
+        client = boto3.client(service_name='bedrock',
```

Comment:
We shouldn't bother even calling this API, if `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are None.